app/main.py

```diff
@@ -1,5 +1,85 @@
+from typing import Union
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+        self,
+        name: str,
+        weight: int,
+        coords: list[int] = None
```

Comment:
If a variable can be None you must show it a type hint
```suggestion
        coords: list[int] | None = None
```